### ISSUE TITLE
Feature request: Recompute Solo SJ output w/o realigning?

### ISSUE THREAD

**Issue by cnk113** [alexdobin/STAR#1321]
Hey Alex,

I was wondering if it was possible to add a simple script similar to your other one to recompute the cell x gene matrix from the BAM, but for the SJ output?

Best,
Chang

**Comment by alexdobin** [comment]
Hi Chang,

this would be a good feature.
Cannot promise the ETA...
I also still need to work on two other bugs you discovered. :)

Cheers
Alex

**Comment by cnk113** [comment]
Yeah, obviously no rush! Understandably there's a lot going on and I'm grateful for your maintenance and enhancements to STAR!

Thanks,
Chang
